chichiwang/exercise-js
# Create component `HTMLBody`, wrap entire document markup in Layout

## Description
Linked to Issue: #126 
The entire document markup will be wrapped in the `Layout` component to allow route data to propagate to the tags within the document head.

To aid this, a `HTMLBody` tag is created, so it can be reused across any number of layouts. This component will render the html, head, and body tags.

The `HTMLBody` tag will be used in layouts to wrap the layout contents. This is done in `app/Layouts/index.tsx`. `app/App` will only render `children`, to ensure that all contents of the app are wrapped in the router.

## Changes
* `app/App/index.tsx` changed to only render `children`, rather than containing the document html, head, and body
* Create component `app/HTMLBody/index.tsx` to render the document html, head, and body
* Wrap contents in the new `HTMLBody` component in `app/Layout/index.tsx`

## Steps to QA
* Pull down and switch to this branch
* Run the app and ensure no issues
* Verify in browser that pages render without issue
